src/ui/src/js/controllers/job/import_systems.js

```diff
@@ -0,0 +1,99 @@
+import template from '../../../templates/import_systems.html'
+
+jobImportSystemsController.$inject = ['$scope', '$rootScope', '$uibModal', 'JobService'];
+
+/**
+ * jobImportSystemsController - Controller for job import.
+ * @param  {Object} $scope            Angular's $scope object.
+ * @param  {Object} $rootScope        Angular's $rootScope object.
+ * @param  {Object} $uibModal         Angular UI's $uibModal object.
+ * @param  {Object} JobService        Beer-Garden's job service.
+ */
+export function jobImportSystemsController( $scope, $rootScope, $uibModal, JobService) {
```

Comment:
I did not use a formatter. I should have. Bad on me. Especially since it's usually me who's complaining about his sort of thing ¯\_(ツ)_/¯ . Will look into how to run one of the javascript prettifiers. It should probably be an action in the package.json so that's it's easy to do in the future.